jukebox_radio/users/models.py

```diff
@@ -32,3 +46,6 @@ def spotify_access_token(self):
         return cipher_suite.decrypt(
             self.encrypted_spotify_access_token.encode("utf-8")
         ).decode("utf-8")
+
+def upload_to_profile_imgs(*args, **kwargs):
+    return f"django-storage/user/imgs" f"{unique_upload(*args, **kwargs)}"
```

Comment:
This is awesome!

I've been using the format `f"django-storage/<app>/<model>/<field>" f"{unique_upload(*args, **kwargs)}"` as the file path. How about the same thing here?

Perhaps `f"django-storage/users/user-profile/profile-imgs" f"{unique_upload(*args, **kwargs)}"`

Reply:
this sounds good! I modeled it after something that you wrote somewhere.